src/analyses/accessAnalysis.ml

```diff
@@ -73,6 +75,17 @@ struct
       ctx.local
     end
 
+  let asm ctx outs ins =
+    if not !ignore_asm then begin
```

Comment:
Should the `asm_is_top` flag impact all analyses? If this check be moved into the `mCP`, so that it only needs to be done once?

Reply:
I think it probably should be at the `FromSpec` level even. And the top-level `asm_is_nop` option should probably be replaced with something more organized like `sem.asm.nop`.